CBSI-PATCHES.md

```diff
@@ -0,0 +1,3 @@
+# CBSi Patches
+
+TODO
```

Comment:
Can we just delete this file until we're ready to put it in?

Reply:
I'd rather leave it as a placeholder.